src/current/_includes/releases/v23.1/v23.1.4.md

```diff
@@ -0,0 +1,67 @@
+
+<h3 id="v23-1-4-{{-site.data.products.enterprise-}}-edition-changes">{{ site.data.products.enterprise }} edition changes</h3>
+
+- Fixed an initialization race condition in [changefeed schema feeds](../v23.1/changefeed-examples.html) that could result in null pointer exceptions. [#104934][#104934]
+
+<h3 id="v23-1-4-sql-language-changes">SQL language changes</h3>
+
+- Users with [`MODIFYSQLCLUSTERSETTING`](../v23.1/security-reference/authorization.html#supported-privileges) will no longer be able to view non-`sql.defaults` [cluster settings](../v23.1/cluster-settings.html). [#104542][#104542]
+
+<h3 id="v23-1-4-operational-changes">Operational changes</h3>
+
+- The [`debug.zip`](../v23.1/cockroach-debug-zip.html) archive now contains the files formerly located at `nodes/*/ranges/*.json` in one file per node, `nodes/*/ranges.json`. [#104248][#104248]
+- The `http-defaults` and `http-servers` sections of the [log config](../v23.1/configure-logs.html) will now accept a `headers` field containing a map of key-value string pairs which will comprise custom HTTP headers appended to every request. Additionally, a `compression` value can now be set to `gzip` or `none` to select a compression method for the HTTP request body. By default, `gzip` is selected. Previous functionality did not compress by default. [#104814][#104814]
+
+<h3 id="v23-1-4-command-line-changes">Command-line changes</h3>
+
+- There is a new log config option for the [`buffering` field](../v23.1/configure-logs.html#log-buffering-for-network-sinks) to [allow buffer output to be formatted as JSON arrays](../v23.1/configure-logs.html#file-logging-format). This is useful for APIs that require JSON array format, such as the Datadog logs API. [#104790][#104790]
+
+<h3 id="v23-1-4-db-console-changes">DB Console changes</h3>
+
+- The tenant dropdown is now hidden for users logged in to fewer than 2 tenants. If the singular tenant is not the system tenant, the dropdown remains visible. [#104819][#104819]
+- Added warning to DB Console overview page when all nodes are running on the new version, but the cluster upgrade is not finalized. [#104874][#104874]
+
+<h3 id="v23-1-4-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug where a transaction retry during the backfill of the [**Jobs** table](../v23.1/ui-jobs-page.html#jobs-table) could result in job rows being missed. [#104757][#104757]
+- Fixed a bug where admin or root user privileges were required to use [`SHOW SYSTEM GRANTS`](../v23.1/show-system-grants.html). [#104732][#104732]
```

Comment:
```suggestion
- Fixed a bug where admin or root user privileges were erroneously required to use [`SHOW SYSTEM GRANTS`](../v23.1/show-system-grants.html). [#104732][#104732]
```
Otherwise this is a bit ambiguous what the bug is.